Cargo.toml

```diff
@@ -8,7 +8,7 @@ include = ["src/**/*", "Cargo.toml", "README.md"]
 readme = "README.md"
 
 [dependencies]
-actix-web = { version = "0.7.19", features = ["tls"] }
+actix-web = { version = "0.7.19", features = ["tls"], default-features = false }
```

Comment:
This is needed because ring conflicts and we do not need the secure cookie anyways.